ui/components/component-library/select-wrapper/select-wrapper.stories.tsx

```diff
@@ -0,0 +1,312 @@
+        <SelectOption value="Option 1">Option 1</SelectOption>
+        <SelectOption value="Option 2">Option 2</SelectOption>
+        <SelectOption value="Option 3">Option 3</SelectOption>
+      </SelectWrapper>
+    </>
+  );
+};
+
+IsMultiSelect.args = {
+  placeholder: 'Select Option',
+  isMultiSelect: true,
+};
+
+IsMultiSelect.storyName = 'isMultiSelect';
+
+export const Placeholder: StoryFn<typeof SelectWrapper> = (args) => {
+  return (
+    <SelectWrapper
+      placeholder={'This is a placeholder'}
+      {...args}
+      triggerComponent={<SelectButton>Demo</SelectButton>}
+    >
+      <SelectOption value="Option 1">Option 1</SelectOption>
+      <SelectOption value="Option 2">Option 2</SelectOption>
+      <SelectOption value="Option 3">Option 3</SelectOption>
+    </SelectWrapper>
+  );
+};
+
+export const IsDisabled: StoryFn<typeof SelectWrapper> = (args) => {
+  return (
+    <SelectWrapper
+      {...args}
+      triggerComponent={<SelectButton>Demo</SelectButton>}
+    >
+      <SelectOption value="Option 1">Option 1</SelectOption>
+      <SelectOption value="Option 2">Option 2</SelectOption>
+      <SelectOption value="Option 3">Option 3</SelectOption>
+    </SelectWrapper>
+  );
+};
+
+IsDisabled.args = {
+  placeholder: 'Select Option',
+  isDisabled: true,
+};
+
+IsDisabled.storyName = 'isDisabled';
+
+export const IsDanger: StoryFn<typeof SelectWrapper> = (args) => {
+  return (
+    <SelectWrapper
+      {...args}
+      triggerComponent={<SelectButton>Demo</SelectButton>}
+    >
+      <SelectOption value="Option 1">Option 1</SelectOption>
+      <SelectOption value="Option 2">Option 2</SelectOption>
+      <SelectOption value="Option 3">Option 3</SelectOption>
+    </SelectWrapper>
+  );
+};
+
+IsDanger.args = {
+  placeholder: 'Select Option',
+  isDanger: true,
+};
+
+IsDanger.storyName = 'isDanger';
+
+export const PopoverProps: StoryFn<typeof SelectWrapper> = (args) => {
+  return (
+    <SelectWrapper
+      {...args}
+      triggerComponent={<SelectButton>Demo</SelectButton>}
+    >
+      <SelectOption value="Option 1">Option 1</SelectOption>
+      <SelectOption value="Option 2">Option 2</SelectOption>
+      <SelectOption value="Option 3">Option 3</SelectOption>
+    </SelectWrapper>
+  );
+};
+
+PopoverProps.args = {
+  placeholder: 'Select Option',
+  popoverProps: {
+    backgroundColor: BackgroundColor.goerli,
+    padding: 4,
+    isPortal: false,
+  },
+};
+
+PopoverProps.storyName = 'popoverProps';
+
+// popoverProps,
+
+export const WIP: StoryFn<typeof SelectWrapper> = (args) => {
+  return (
+    <>
+      <SelectWrapper
+        {...args}
+        onBlur={() => console.log('blur')}
+        triggerComponent={<SelectButton>Uncontrolled Example</SelectButton>}
+      >
+        <SelectOption value="Option 1">Option 1</SelectOption>
+        <SelectOption value="Option 2">Option 2</SelectOption>
+        <SelectOption value="Option 3">Option 3</SelectOption>
+      </SelectWrapper>
+    </>
+  );
+};
```

Comment:
I see this is still a WIP so probably don't need me to comment here that its not closing on blue


https://github.com/MetaMask/metamask-extension/assets/8112138/546b8697-6e0d-4ba5-b8cc-297041e72bc2